package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
     "build": "eslint src && pulp build -- --censor-lib --strict",
-    "test": "pulp build -I test"
+    "test": "pulp test -- --censor-lib --strict"
```

Comment:
I'm not sure we want `--strict`, because tests ought to be able to contain warnings if they're useful in the test suite. That's different from production code, which shouldn't have any warnings.